static_files/validator/validator_setup.sh

```diff
@@ -74,6 +94,11 @@ setup_validator() {
     heimdalld-v2 init --home "${cl_validator_config_path}" --chain-id "${CL_CHAIN_ID}" "${id}" 2>"${cl_validator_config_path}/init.out"
     # HOTFIX: heimdalld-v2 outputs info and debug lines before the json object.
     node_id="$(cat ${cl_validator_config_path}/init.out | sed '/.DBG\|INF./d' | jq --raw-output '.node_id')"
+
+    # Replace "comet/PubKeySecp256k1Uncompressed" with "cometbft/PubKeySecp256k1eth" in priv_validator_key.json
+    sed -i 's/comet\/PubKeySecp256k1Uncompressed/cometbft\/PubKeySecp256k1eth/g' "${cl_validator_config_path}/config/priv_validator_key.json"
+    # Replace "comet/PrivKeySecp256k1Uncompressed" with "cometbft/PrivKeySecp256k1eth" in priv_validator_key.json
+    sed -i 's/comet\/PrivKeySecp256k1Uncompressed/cometbft\/PrivKeySecp256k1eth/g' "${cl_validator_config_path}/config/priv_validator_key.json"
```

Comment:
This is a temporary fix for the discrapency of the version between [pos-validator-config-generator](https://github.com/0xPolygon/kurtosis-polygon-pos/blob/main/README.md?plain=1#L284) and latest version of [heimdall-v2](https://github.com/0xPolygon/heimdall-v2). Once the version of pos-validator-config-generator is bumped, we can remove this and use `cometbft/PubKeySecp256k1eth` from the beginning.